package.json

```diff
@@ -55,24 +55,24 @@
     "@commitlint/cli": "8.1.0",
     "@commitlint/config-conventional": "8.1.0",
     "@percy/puppeteer": "1.0.8",
-    "@testing-library/cypress": "4.0.4",
-    "@testing-library/react": "8.0.4",
+    "@testing-library/cypress": "4.0.5",
+    "@testing-library/react": "8.0.6",
     "@types/common-tags": "1.8.0",
     "@types/enzyme": "3.10.3",
-    "@types/graphql": "14.2.2",
+    "@types/graphql": "14.2.3",
     "@types/history": "4.7.2",
     "@types/jest": "24.0.15",
     "@types/lodash": "4.14.136",
     "@types/moment-timezone": "0.5.12",
-    "@types/node": "10.14.13",
+    "@types/node": "12.6.8",
```

Comment:
We use node 10, so we should keep it pinned to v10

Reply:
Yeah I noticed and downgraded already again.